packages/material-ui/src/Box/styleFunction.js

```diff
@@ -72,7 +72,8 @@ const getThemeValue = (prop, value, theme) => {
   return { [prop]: value };
 };
 
-export const styleFunctionSx = (styles, theme) => {
+export const styleFunctionSx = (props = {}) => {
```

Comment:
How do these runtime changes relate to the docs? Would be nice to have tests for them so that we can decipher later why these changes were necessary.

Reply:
Will extract them in a dedicated PR